src/reduced_basis/rb_data_deserialization.C

```diff
@@ -329,14 +327,12 @@ void load_parameter_ranges(RBParametrized & rb_evaluation,
         discrete_parameter_values[discrete_names[i]] = values;
       }
   }
-#ifdef LIBMESH_ENABLE_EXCEPTIONS
-  catch (std::exception & e)
+  libmesh_catch (std::exception & e)
   {
     libmesh_error_msg("Error loading parameter ranges from capnp reader.\n"
                       "This usually means that the training data either doesn't exist or is out of date.\n"
                       "Detailed information about the error is below:\n\n" << e.what() << "\n");
```

Comment:
I chatted with @roystgnr, and he's attaching or already attached a "Test No Exceptions" recipe. I think we can't refer to `e.what()` here when exceptions are disabled, this is the same error that @pbehne ran into.

Reply:
But we're foiled in our testing by `!LIBMESH_HAVE_CAPNPROTO` - I'll fix this regardless.

Reply:
Yeah, I should just drop this commit.  Looks like the existing code was correct and I shouldn't risk breaking it to save a couple preprocessor lines.